commands.sh

```diff
@@ -24,24 +24,26 @@ sudo systemctl enable apache2
 
 # Create a directory for your Apache container
 sudo mkdir my-apache-container
-cd my-apache-container
+sudo cd my-apache-container
 
 # Create a Dockerfile for your Apache container
-sudo cat <<EOL > my-apache-container/Dockerfile
+sudo cat <<EOL > Dockerfile
 # Use the official Apache HTTP Server image as the base image
 FROM httpd:latest
```

Comment:
You should have to use apache lastest specfic versions to ensure that it will remain same during each deployment and image creation